editor/import/3d/resource_importer_scene.cpp

```diff
@@ -2168,6 +2168,16 @@ Node *ResourceImporterScene::_generate_meshes(Node *p_node, const Dictionary &p_
 			} break;
 		}
 
+		{
+			List<StringName> meta_list;
+			p_node->get_meta_list(&meta_list);
+			for (List<StringName>::Element *element = meta_list.front(); element; element = element->next()) {
```

Comment:
```suggestion
			for (List<StringName>::Element *E = meta_list.front(); E; E = E->next()) {
```
We normally name it so

Reply:
Ah ok, great!